src/components/interfaces/MOBILE_API.xml

```diff
@@ -1,5364 +1,5408 @@
-	</param>
-    <param name="carModeStatus" type="CarModeStatus">
-      <description>References signal "CarMode". See CarMode.</description>
-	</param>
-    <param name="powerModeStatus" type="PowerModeStatus">
-      <description>References signal "PowerMode". See PowerMode.</description>
-	</param>
-  </struct>
-
-  <struct name="MyKey">
-    <param name="e911Override" type="VehicleDataStatus">
-      <description>Indicates whether e911 override is on.  References signal "MyKey_e911Override_St". See VehicleDataStatus.</description>
-	</param>
-  </struct>
-
-<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
-<!-- / Ford Specific Data Items -->
-<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
-
-  <enum name="VehicleDataResultCode">
-    <description>Enumeration that describes possible result codes of a vehicle data entry request.</description>
-    <element name="SUCCESS" internal_name="VDRC_SUCCESS">
-      <description>Individual vehicle data item / DTC / DID request or subscription successful</description>
-    </element>
-    <element name="TRUNCATED_DATA" internal_name="VDRC_TRUNCATED_DATA">   
-      <description>DTC / DID request successful, however, not all active DTCs or full contents of DID location available</description>
-    </element>
-    <element name="DISALLOWED" internal_name="VDRC_DISALLOWED">
-      <description>This vehicle data item is not allowed for this app by Ford.</description>
-    </element>
-    <element name="USER_DISALLOWED" internal_name="VDRC_USER_DISALLOWED">
-      <description>The user has not granted access to this type of vehicle data item at this time.</description>
-    </element>
-    <element name="INVALID_ID" internal_name="VDRC_INVALID_ID">
-      <description>The ECU ID referenced is not a valid ID on the bus / system.</description>
-    </element>
-    <element name="VEHICLE_DATA_NOT_AVAILABLE" internal_name="VDRC_DATA_NOT_AVAILABLE">
-      <description>The requested vehicle data item / DTC / DID is not currently available or responding on the bus / system.</description>
-    </element>
-    <element name="DATA_ALREADY_SUBSCRIBED" internal_name="VDRC_DATA_ALREADY_SUBSCRIBED">
-      <description>The vehicle data item is already subscribed.</description>
-    </element>
-    <element name="DATA_NOT_SUBSCRIBED" internal_name="VDRC_DATA_NOT_SUBSCRIBED">
-      <description>The vehicle data item cannot be unsubscribed because it is not currently subscribed.</description>
-    </element>
-    <element name="IGNORED" internal_name="VDRC_IGNORED">    
-      <description>The request for this item is ignored because it is already in progress.</description>
-    </element>
-  </enum>
-
-  <struct name="TireStatus">
-      <description>The status and pressure of the tires.</description>
-
-    <param name="pressureTelltale" type="WarningLightStatus">
-      <description>Status of the Tire Pressure Telltale. See WarningLightStatus.</description>
-	</param>
-    <param name="leftFront" type="SingleTi
```

Comment:
@LevchenkoS There is something wrong with the diff in this file, most of the lines are the same so, it is hard to tell what exactly changed (in scope of this commit), please check if it possible to make diff here more readable, or maybe drop this file change and add its changes manually.